libsplinter/src/biome/oauth/store/mod.rs

```diff
@@ -21,7 +21,8 @@ pub(in crate::biome) mod diesel;
 mod error;
 pub(in crate::biome) mod memory;
 
-pub use error::InvalidStateError;
+pub use crate::error::InvalidStateError;
```

Comment:
I don't think we should pub use crate::error:: things to local modules.

Reply:
Oops, good catch; forgot to remove the `pub`. Updated.